src/features/grading/GradingTypes.ts

```diff
@@ -9,6 +11,24 @@ import {
 } from '../../commons/assessment/AssessmentTypes';
 import { Notification } from '../../commons/notificationBadge/NotificationBadgeTypes';
 
+export enum ColumnFields {
+  assessmentName = 'assessmentName',
+  assessmentType = 'assessmentType',
+  studentName = 'studentName',
+  studentUsername = 'studentUsername',
+  groupName = 'groupName',
+  progressStatus = 'progressStatus',
+  gradingStatus = 'gradingStatus',
+  xp = 'xp',
+  actionsIndex = 'actionsIndex'
+}
+
+export enum SortStates {
+  ASC = 'sort-asc',
+  DESC = 'sort-desc',
+  NONE = 'sort'
```

Comment:
If you're using an enum, please use `SortStates.NONE` above (in the backend saga) instead of the raw string value.